gatsby-config.js

```diff
@@ -1,5 +1,41 @@
 module.exports = {
   plugins: [
+    {
+      resolve: `gatsby-source-apiserver`,
+      options: {
+        enableDevRefresh: true,
+        name: "assets",
+        url: "https://api.adoptopenjdk.net/v3/assets",
```

Comment:
It would probably be good to make the URL a parameter so we can change it easily if needed

Reply:
Yes I'll probably do that before merge.